apps/modernization-api/src/test/resources/features/patient/profile/PatientProfilePhoneChange.feature

```diff
@@ -41,3 +41,10 @@ Feature: Patient Profile Phone Changes
     And I have the authorities: "FIND-PATIENT" for the jurisdiction: "ALL" and program area: "STD"
     Then I am unable to remove a patient's phone
     And a patient event is not emitted
+    
+   Scenario: I can update a patient's existing phone
```

Comment:
These are tests for changing adding, updating, and removing Phone Numbers for an existing patient.  I think this story is meant for removing duplicates in Patient Search Results.

Reply:
sure I will test it in isolation as mentioned in the quote reply.

Reply:
@adamloup-enquizit  updated.